### DISCUSSION TITLE
2023-05-24 Weekly Meeting 108 Notes/Agenda

### DISCUSSION THREAD

**Discussion by funksoup** [omigroup/omigroup#392] [General]
Weekly Meeting 108 is happening on Wednesday, May 24, 2023 from 9:00 AM PST (16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

UPDATES:

MSF-delegates:

- REMINDER: There is a board election coming up - it's important that we/OMI have a representative on the MSF board, so we can continue to be a voice for independent creators, small orgs, open source and community/grassroots orgs. Elections will be in a few weeks but we can start preparing our community for that now. (There are some documents we will need to fill out). **We want to put one of us on the board (i.e. @funksoup)**. 

- Interoperability issue to bring to the attention of MSF and industry in general (via @aaronfranke):
-- There are competing standards between OMI's [OMI_collider](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_collider) and [OMI_physics_body](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_physics_body) glTF extensions and Microsoft's physics https://github.com/KhronosGroup/glTF/pull/2257.
-- Evo will interview @aaronfranke and then post the video so that we can get more engagement on this issue.
-- How can we get more engagement? Email the MSF general list (but make the email shorter - make ask clear)

-- Issue is described in detail in this [OMI_collider and OMI_physics_body pull request](https://github.com/KhronosGroup/glTF/pull/2258) and there's a bulleted list of specific differences and impacts. 

Media group: 

GLTF group:

WORLD HOPS:
Status of metatraversal world hop? Real world/virtual world hybrid event for [AWE](https://www.awexr.com/)
AWE conference is May 31-June 2 in Santa Clara, CA. If we can get some folks who will be onsite at AWE to help facilitate, we could do an OMI metatraversal world hop, similar to what we did at the OMI 100th meeting anniversary event. 

We could visit / meet in these spaces (via @madjin): 
https://hyperfy.io/msf
https://hyperfy.io/neonstreet
https://hyperfy.io/omi
https://framevr.io/omi

**Comment by mrmetaverse** [comment]
We want to send this to people working on standards. We are at risk of confusing physics standards. There are two competing physics standards between OMI and MSFT. We want to merge these. We are asking for OMI communal input to add your voice and input to help ensure more voices are heard. https://hedgedoc.openmv.org/s/wvLuUfkBg#

<img width="594" alt="Screenshot 2023-05-24 at 12 21 42 PM" src="https://github.com/omigroup/omigroup/assets/63426722/c1914273-878b-42af-800d-fce40d6f6c8f">

**Reply by mrmetaverse** [reply]
I have added my personal thoughts to this MSF PR (Metaverse Standards Forum Pull Request) thread: https://github.com/KhronosGroup/glTF/pull/2258#issuecomment-1561548415

**Comment by mrmetaverse** [comment]
"these conversations often open doors that we don't see" <3

**Reply by mrmetaverse** [reply]
"firewalls be damned" 😂

**Comment by mrmetaverse** [comment]
## Media 

- evo to post a streamyard link shortly for Aaron and anyone else who needs video content today 
- keyframe made a quick 1 min edit to cover the meta traversal portal crawl that we can share out: https://youtu.be/FiaVs-57lUg

**Comment by mrmetaverse** [comment]
## glTF 
- see above for updates about the physics spec conflict merge effort 
- working on getting physics materials working, Robert to spearhead that 
- need to fix KHR Audio 
- noticed a rising trend in godot chatter about 3d models, colliders, and physics standards. People are starting to realize that if they could put the physics in blender and pull it into a game engine and _it just works_ that would be great.

**Comment by mrmetaverse** [comment]
## OMI portal crawl 

On Friday we had a portal crawl! Hopping around was fun. There were some silly firewall issues on Google internet for some. It was cool being able to jump in and out from bases like ReadyPlayerMe, and Nowhere. 

The audio channels were a little challenging. Having a single unified audio stream instead of using each individual worlds audio may have been easier. A cohesive audio track during the live stream could help provide more stability.

**Comment by mrmetaverse** [comment]
## Open Collective 

- #361 blocked, reached out to OC customer service. 
- the blocker: It won't let me reference OMI in a new filing, or convert the existing OMI profile to the new 501c3. I am asking if this is normal behavior. Do I need to start entirely new or can I repurpose our existing OMI (since we have donors already, etc.)
